install_files/securedrop-keyring/DEBIAN/preinst

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+
+# Solution adapted from DKG's work on `deb.torproject.org-keyring`.
+# In SecureDrop versions before 0.3.10, the Ansible playbooks used
+# the apt_key module to add the signing key to /etc/apt/trusted.gpg.
+# It's cleaner to use the trusted.gpg.d subdirectory, since we can
+# update that trivially in future versions of the keyring package.
+#
+# Therefore let's clean up prior versions of the key installed
+# to the general apt keyring, to ensure we only have one signing key
+# installed for authenticating securedrop-related packages.
+
+if [ -e /etc/apt/trusted.gpg ] && which gpg >/dev/null; then
+   (
+   h="$(mktemp -d)"
+   trap "rm -rf '$h'" EXIT
+
+   if gpg --homedir="$h" \
+          --batch --no-tty --no-default-keyring --keyring /etc/apt/trusted.gpg \
+          --list-key 0xBFFDF68849BC8719F1BE0191321A7B10AE2C8E02 > /dev/null 2>&1 ; then
+      gpg --homedir="$h" \
+          --batch --no-tty --no-default-keyring --keyring /etc/apt/trusted.gpg \
+          --no-auto-check-trustdb \
+          --delete-key 0xBFFDF68849BC8719F1BE0191321A7B10AE2C8E02 || true
```

Comment:
You're right! That's _not_ the key we want to remove—that's a testing key used temporarily on the staging apt server. Updated with the currently active fingerprint `0xB89A29DB2128160B8E4B1B4CBADDE0C7FC9F6818`, and rebased for clean commit history. 

New commit implementing correct fingerprint is cd36f6909c4abbcf5ec950fe174f9cf70731914c (was 26559ee334e85e466f5bd90ae2baca94ccc83d12).